src/Runners/EchoFakeRunner.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sergey
+ * Date: 30.01.17
+ * Time: 12:07
+ */
+
+namespace AdamBrett\ShellWrapper\Runners;
+
+
+use AdamBrett\ShellWrapper\Command\CommandInterface;
+use AdamBrett\ShellWrapper\Runners\FakeRunner;
+
+class EchoFakeRunner extends FakeRunner
```

Comment:
I think this should just be called `DryRunner`. I don't think that it matters that it extends/is Fake